README.md

```diff
@@ -36,9 +36,9 @@ You'll be prompted a few questions to help this cookiecutter template to scaffol
 Option | Description
 ------------------------------------------------- | ---------------------------------------------------------------------------------
 `include_apigw` | Includes sample code for API Gateway Proxy integration for Lambda and a Catch All method in SAM as a starting point
-`include_xray` | Includes both sample code for getting started with AWS X-Ray and adds necessary permission and `Tracing` to your function
+`include_lambda_powertools` | Includes both sample code for getting started with [AWS Lambda Powertools](https://github.com/awslabs/aws-lambda-powertools-python/) and adds necessary permission and `Tracing` to your function
```

Comment:
**what if we enable this by default? i.e. remove the selection** They address operational best practices, and it's something they'd use one way or the other. API GW and Safe deployments, on the other hand, might not be always wanted.

Reply:
Sure, I can do it